### PR TITLE
Quickstarts

### PR DESCRIPTION
**Pull request by khorne3**
* [Homepage](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/quickstart/)
* [Linux/macOS](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/quickstart/quickstarts/linux-macos)
* [Windows](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/quickstart/quickstarts/windows)

### CODE REVIEW

File: quickstarts/linux-macos.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Begin by selecting the operating system you are using (in this case, select macOS); this will update the command-line code samples ShiftLeft provides you in the following steps.
+
+Next, install the ShiftLeft CLI. You can do this in one of two ways:
+
+1. Downloading the ShiftLeft CLI and manually adding the CLI to your system path.
+
+2. Running the provided command-line command to download and adjust the permissions for the CLI:
+
+```text
```
Comment (vlad-alexandru-ionescu) [inline quickstarts/linux-macos.md]: Use bash syntax highlighting perhaps?

```suggestion
```bash
```

BTW - should we prefix commands with the `$` prompt or not?

File: quickstarts/linux-macos.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+31.85 MB / 31.85 MB [===================================] 100.00% 10.73 MB/s 2s
+
+Saved config to shiftleft.json
+... Done. Submitted for analysis
+Wait for 5-10 minutes and load the following URL in your browser:
+https://www.shiftleft.io/apps/HelloShiftLeft?organizationId=d64f2e4d-0d32-4c0b-bdae-891f693e2399
+```
+
+Per the instructions printed to the Terminal/Bash, open up the URL provided after 5-10 minutes have elapsed. This brings you to the Dataflows page; in the top right-hand corner, click Vulnerabilities to see the issues present in your code.
```
Comment (vlad-alexandru-ionescu) [inline quickstarts/linux-macos.md]: Perhaps this URL should take you straight to vulnerabilities view. Mind opening an issue in Product, for @mirho , to change the default landing for this URL?

File: quickstarts/windows.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+![Dashboard Instructions Page](img/add-app.png)
+
+You can always return to this page at a later date by clicking **Add App** in the Dashboard.
+
+As a Windows user, the only thing you need to do on this page is to make a note of the information provided to associate the CLI with your ShiftLeft account:
+
+```text
+sl auth --org "YOUR_ORG_ID" --token "YOUR_ACCESS_TOKEN"
+```
```
Comment (vlad-alexandru-ionescu) [inline quickstarts/windows.md]: This step would be done after the installation. Perhaps this should be moved further down the page.

File: quickstarts/linux-macos.md (hunk 2 above)
Comment (khorne3) [reply quickstarts/linux-macos.md]: https://github.com/ShiftLeftSecurity/product/issues/3536

File: quickstarts/linux-macos.md (hunk 1 above)
Comment (khorne3) [reply quickstarts/linux-macos.md]: Yes to both

**Comment by khorne3** [comment]
Hey @vlad-alexandru-ionescu -- I've added your suggestions, updated the navigation a bit, and added some context + quickstart links to the homepage. If you have no additional comments, I think this is ready to go.

**Pull request merged**